membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -155,6 +155,7 @@ object AccountDetails {
         ),
       ) ++
         regNumber.fold(Json.obj())({ reg => Json.obj("regNumber" -> reg) }) ++
+        billingCountry.fold(Json.obj())({ bc => Json.obj("billingCountry" -> bc.name)}) ++
```

Comment:
Is name definitely what we want here, not `alpha2`?

Reply:
Yes, this makes it similar to `deliveryAddress.country` pulled in from Salesforce. Manage-frontend uses the country name to evaluate the Stripe token: https://github.com/guardian/manage-frontend/blob/42a23b000f95c4d5f273e3d807cfce2c7c1c6f5d/client/stripe.ts#L10